fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -215,7 +215,9 @@ const RollCallOpened = () => {
         setVisibility={setInputModalIsVisible}
         title={STRINGS.roll_call_modal_add_attendee}
         description={STRINGS.roll_call_modal_enter_token}
-        onConfirmPress={addAttendeePopTokenAndShowToast}
+        onConfirmPress={(token: string) =>
+          addAttendeePopTokenAndShowToast(`{"pop_token":"${token}"}`)
```

Comment:
```suggestion
          addAttendeePopTokenAndShowToast(JSON.stringify({ pop_token: token }))
```

Detail: to make it more clear that you're doing a JSON

Reply:
Good catch.